src/locales/de/translation.json

```
@@ -100,10 +101,23 @@
     "status.CREATED.advisor.tooltip": "Berateraccount aktiv",
     "status.ACTIVE.tooltip": "Aktiv",
     "status.INACTIVE.tooltip": "Inaktiv",
+    "status.INVITED.tooltip": "Eingeladen / Einrichtung ausstehend",
+    "status.ABSENT.tooltip": "Abwesend",
+    "status.DISABLED.tooltip": "Deaktiviert",
+    "users.table.lastUpdated": "Zuletzt aktualisiert",
+    "users.table.expand": "Zeilen erweitern",
+    "users.table.collapse": "Zeile einklappen",
+    "users.allUsers": "Alle Nutzer",
```

Biome (2.4.16)

[error] The key users.allUsers was already declared.

(lint/suspicious/noDuplicateObjectKeys)